DEPLOYING.md

```diff
@@ -69,75 +69,71 @@ contains step-by-step deployment instructions using opeNode's command line inter
+ 
+5. Edit the Dockerfile. You must add the line `ENV BASE_URL=http://YOUR_SITE_NAME.openode.io/` before you deploy or it won't be able to build. You can find your site name on the openode website. You also must add `ENV PORT=80` as it is not included in the samples Dockerfile though it is included in the one that is generated when you type openode template. 
+
+    So add lines:
+
+    ```
+    ENV PORT=80
+    ENV BASE_URL=http://YOUR_SITE_NAME.openode.io/
+    ```
+
+    after the `WORKDIR /opt/mre` line to your Dockerfile before you deploy.
+
+    Save and close the Dockerfile.
+ 
 6. Deploy your app
-
-To start a deployment, run this command:
-```
-openode deploy
-```
-This step copies your project to your opeNode instance, installs npm dependencies, and then starts it running.
-The first time you deploy your MRE SDK app, you may see an error message that starts with this:
-```
-dtrace-provider@0.8.7 install /opt/app/node_modules/dtrace-provider
-> node-gyp rebuild || node suppress-error.js
-
-gyp ERR! configure error
-gyp ERR! stack Error: Can't find Python executable "python", you can set the PYTHON env variable.
-```
-If you see this error, run `openode deploy` one more time and it should clear up. If you see this
-error again, please [open a GitHub issue here](https://github.com/Microsoft/mixed-reality-extension-sdk/issues/new).
-
-Once your deployment succeeds, you're ready to connect from AltspaceVR! For instructions
-on instantiating your app within AltspaceVR, checkout the [sample repo's README.md](
+ 
+    To start a deployment, run this command:
+    ```
+    openode deploy
+    ```
+    This step copies your project to your opeNode instance, installs npm dependencies, and then starts it running.
+    The first time you deploy your MRE SDK app, you may see an error message that starts with this:
+    ```
+    dtrace-provider@0.8.7 install /opt/app/node_modules/dtrace-provider
+    > node-gyp rebuild || node suppress-error.js
+ 
+    gyp ERR! configure error
+    gyp ERR! stack Error: Can't find Python executable "python", you can set the PYTHON env variable.
+    ```
+    If you see this error, run `openode deploy` one more time and it should clear up. If you see this
+    error again, please [open a GitHub issue here](https://github.com/Microsoft/mixed-reality-extension-sdk/issues/new).
+ 
+Once your deployment succeeds, you're ready to connect from AltspaceVR! 
+ 
+Open the AltspaceVR app, go to your world editor, press basics, then press SDK apps, then type wss://YOUR_SITE_NAME.openode.io/.
+Note you must use wss:// or it will not connect. 
+ 
+For instructions on instantiating your app within AltspaceVR, checkout the [sample repo's README.md](
 https://github.com/Microsoft/mixed-reality-extension-sdk-samples/blob/master/README.md)
 
+When you run your MRE in Altspace make sure you have the openode website opened on your browser or it will not deploy at times.
```

Comment:
Have you verified this? I would be very surprised if having the website open made any difference.

Reply:
okay this is probably false just checked